pkg/config/setup/config.go

```diff
@@ -716,6 +716,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("admission_controller.inject_config.socket_path", "/var/run/datadog")
 	config.BindEnvAndSetDefault("admission_controller.inject_config.trace_agent_socket", "unix:///var/run/datadog/apm.socket")
 	config.BindEnvAndSetDefault("admission_controller.inject_config.dogstatsd_socket", "unix:///var/run/datadog/dsd.socket")
+	config.BindEnvAndSetDefault("admission_controller.inject_config.type_socket_volumes", false)
```

Comment:
since we know that we want this config option to be use by some customers for specific usecase, could you add it in the config doc template: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml 🙇

Reply:
Added a new commit with this :+1: